ETL/Pipeline/conftest.py

```diff
@@ -0,0 +1,47 @@
+"""Contains all the fixtures used in the test extract file"""
+import pytest
+
+
+@pytest.fixture(name="fake_headers")
+def fixture_fake_headers() -> dict:
+    """Example data for headers"""
+    return {'User-Agent': 'test'}
+
+
+@pytest.fixture(name="fake_product_data")
+def fixture_fake_product_data() -> dict:
+    """Example data for a product"""
+    return {
+        "product_id": 12345,
+        "previous_price": 100,
+        "url": "www.asos.com/shoes",
+        "product_name": "adidas Originals Samba LT trainers"}
+
+
+@pytest.fixture(name="fake_product_list")
+def fixture_fake_product_list() -> list[dict]:
+    """Example data of a list of products"""
+    return [{
+        'product_id': 205759745,
+        'current_price': '',
+        'previous_price': 85,
+        'url': "https://www.asos.com/adidas-originals/adidas-originals-gazelle-trainers-in-\
+            white-and-blue/prd/205759745#ctaref-we%20recommend%20carousel_11&featureref1-we%20recommend%20pers",
+        'product_name': 'adidas Originals Gazelle trainers in white and blue White'
+    }, {
+        'product_id': 206059258,
+        'current_price': '',
+        'previous_price': 100,
+        'url': "https://www.asos.com/adidas-originals/adidas-originals-samba-lt-trainers-in-yellow\
+            -and-green/prd/206059258#ctaref-we%20recommend%20carousel_12&featureref1-we%20recommend%20pers",
```

Comment:
This should be 
```suggestion
        'url': "https://www.asos.com/adidas-originals/adidas-originals-samba-lt-trainers-in-yellow\
-and-green/prd/206059258#ctaref-we%20recommend%20carousel_12&featureref1-we%20recommend%20pers",
```

`\` starts the string at the start of the next line so it may not work as you think

Reply:
Removed the whitespaces.

Reply:
Add another `\` so each line is below 100 characters